YurisCodingClub/dev-cv
# Add empty `CV.js` component in DIN A4 format to store values (#22)

## Description



- Create and add empty `CV.js` component of size ` 2480 x 3508 `
- To keep the ratio smaller, divide both `height` and `weight` by 4 
- `height=2480/4=620` and 
- `weight=3508/4=877`
- The new `height` and `weight` is `620 x 877`
- Give a `2px solid black` border for the visibility of the component

## Related Issue


Closes #22

## Acceptance Criteria



- [x] Create a component called CV.js and save it in a folder called components in the src directory
- [x] The component is in format of DIN A4 (ratio, of course it can be smaller)
- [x] Additional styling: Only a border is added to showcase the component
- [x] The new component is imported and displayed in the App.js component

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
| ✓| :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before




The below of `Education` component is empty before adding the `CV` component<br>
![image](https://user-images.githubusercontent.com/47440165/194778056-eed0e9f9-9ef6-4ba3-9182-cad113d0b8b7.png)



### After




Place the `CV` component below the `Education` component <br>
![image](https://user-images.githubusercontent.com/47440165/194777971-1097296a-0b36-4851-b88c-56dc4fba36cd.png)


## Testing Steps / QA Criteria



- Please check `dev-cv/src/components/CV.js` for the `CV` component
- Import the `CV` component in `dev-cv/src/App.js`
- You can also check the deploy preview here: https://deploy-preview-25--dev-cv-ycc.netlify.app/
